src/NuGet/Microsoft.Orleans.OrleansTelemetryConsumers.Counters.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Orleans.OrleansTelemetryConsumers.Counters</id>
```

Comment:
A package for counter control already exists.  Why not have this in that package?  This would lead to two interdependent counter packages we'd need to maintain.